sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Constructors.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.spark.sql.catalyst.expressions.Expression
 import org.apache.spark.sql.catalyst.expressions.codegen.CodegenFallback
 import org.apache.spark.sql.catalyst.util.GenericArrayData
 import org.apache.spark.sql.sedona_sql.UDT.GeometryUDT
-import org.apache.spark.sql.types.{DataType, Decimal}
+import org.apache.spark.sql.types._
```

Comment:
If this change is not necessary, do not commit this change. You can replace the "_" with the original content

Reply:
Fixed